sdk/search/Azure.Search/src/DocumentsClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Search
+{
+    [CodeGenClient("Documents")]
```

Comment:
The generated clients are public by default but `CodeGenClient` attribute can be used to rename/make them internal.

Reply:
This is definitely fine for our next couple of previews.  We should see if we can avoid the code size increase for GA.